client/signup/steps/jpo-contact-form/style.scss

```diff
@@ -0,0 +1,121 @@
+.jpo__contact-form-wrapper {
+	margin-left: auto;
+	margin-right: auto;
+	display: flex;
+	max-width: 325px;
+}
+
+.jpo__contact-form-wrapper .card {
+	max-width: 320px;
+	text-align: center;
+}
+
+.jpo-contact-form__description {
+	font-size: 1.25rem;
+	color: $gray;
+}
+
+.jpo-contact-form__image {
+	padding:15px;
+}
+
+.jpo-contact-form__choice-copy {
+	padding: 15px;
+	border-top: 1px solid transparentize( lighten( $gray, 20% ), .5 );
+}
+
+.jpo-contact-form__choice.card {
+	width: 46%;
+	display: inline-block;
+	vertical-align: top;
+	text-align: center;
+	padding: 0;
+
+	@include breakpoint( "<480px" ) {
+	width: 100%;
+	text-align: left;
+	}
+
+	&:hover,
+	&.is-selected {
+		box-shadow: 0 0 0 1px $gray, 0 2px 4px lighten( $gray, 20 );
+	}
+
+	svg {
+		width: 94%;
+		height: auto;
+		margin-bottom: 1rem;
+	}
+
+	.button {
```

Comment:
We can remove almost all of these styles. The `color` would be the only exception.